server/routes/api/users.js

```diff
@@ -6,12 +6,6 @@ import AuthMiddleware from '../../utils/middleware/AuthMiddleware';
 
 const router = Router();
 
-router.get(
```

Comment:
Why did you delete this route?

Reply:
Because it retrieve information of the user and that is already implemented in the get profile route

Reply:
The get user route is not being used anyway in the project. It just retrieves the user and returns a user. This is part of what was reducing test coverage

Reply:
alright